library/styleguideComponents/ComponentsList/contactusForm.js

```diff
@@ -0,0 +1,189 @@
+                <Row>
+                    <Field
+                        name="company"
+                        component={SmallInput}
+                        label="Company Name"
+                        placeholder="Enter your company name"
+                        disabled={isLoading}
+                    />
+                    <Field
+                        name="jobtitle"
+                        component={SmallInput}
+                        label="Job Title"
+                        placeholder="Enter your job title"
+                        disabled={isLoading}
+                    />
+                </Row>
+                <Field
+                    required
+                    component={Textarea}
+                    name="message"
+                    className="rainbow-m-bottom_large"
+                    label="Message"
+                    placeholder="What else should we know?"
+                    disabled={isLoading}
+                />
+                <Footer>
+                    <Field name="recaptcha" component={ReCaptcha} siteKey={RECAPTCHA_APIKEY} />
+                    <SendButton variant="brand" isLoading={isLoading} type="submit">
+                        Send
+                        <ArrowRight className="rainbow-m-left_small" />
+                    </SendButton>
+                </Footer>
+            </FormContainer>
+        </Dropdown>
+    );
+}
+
+const validate = values => {
+    const { firstname, lastname, email, message, recaptcha } = values;
+    const errors = {};
+    if (!firstname) {
+        errors.firstname = 'Looks like you forgot your first name.';
+    }
+    if (!lastname) {
+        errors.lastname = 'Looks like you forgot your last name.';
+    }
+    if (!email) {
+        errors.email = 'Looks like you forgot your email.';
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(email)) {
+        errors.email = 'Invalid email.';
+    }
+    if (!message) {
+        errors.message = 'Looks like you forgot leave a message.';
+    }
+    if (!recaptcha) {
+        errors.recaptcha = 'Are you a robot?';
+    }
+    // eslint-disable-next-line no-debugger
+    return errors;
+};
+
+ContactUsForm.propTypes = {
+    closeForm: PropTypes.string,
+    sendMessage: PropTypes.func.isRequired,
+    handleSubmit: PropTypes.func,
+    reset: PropTypes.func,
+    onSuccessMessageSent: PropTypes.func,
+};
+
+ContactUsForm.defaultProps = {
+    closeForm: null,
+    handleSubmit: () => {},
+    reset: () => {},
+    onSuccessMessageSent: () => {},
+};
+
+const mapStateToProps = state => {
+    return { contactUs: state.contactUs, closeForm: state.message.message };
+};
+
+const mapDispatchToProps = dispatch => {
+    return bindActionCreators(
+        {
+            sendMessage,
+        },
+        dispatch,
+    );
+};
+
+const ReduxFormContactUs = reduxForm({
+    form: 'contactUs',
+    validate,
+    touchOnBlur: false,
+})(ContactUsForm);
+
+export default connect(
```

Comment:
instead of using connect, bindActionCreators, mapDispatchToProps and mapStateToProps use the redux hooks useDispatch and useSelector which are much cleaner and simpler, maybe you need to update redux related packages